benchmarks/test_cib.py

```diff
@@ -18,7 +18,7 @@ def test_cibcl():
     cM = ccl.halos.ConcentrationDuffy08(mdef)
     nM = ccl.halos.MassFuncTinker10(cosmo, mdef, norm_all_z=True)
     bM = ccl.halos.HaloBiasTinker10(cosmo, mdef)
-    hmc = ccl.halos.HMCalculator(cosmo, nM, bM, mdef)
+    hmc = ccl.halos.HaloModel(cosmo, nM, bM, mdef)
```

Comment:
Absolutely no, please. The transition will already be painful enough for users, this PR is already enough of a nightmare, and I don't see any good reason for doing this